l10n_it_delivery_note/models/stock_picking.py

```diff
@@ -433,9 +433,12 @@ def write(self, vals):
 
         return res
 
-    def _create_backorder(self):
-        """When we make a backorder of a picking the delivery note lines needed
-        to be updated otherwise stock_delivery_note_line_move_uniq constraint is raised"""
-        backorders = super()._create_backorder()
+    def _create_backorder(self, backorder_moves=None):
```

Comment:
In `14.0` la firma del metodo non prevedere parametri aggiuntivi (Rif. https://github.com/odoo/odoo/blob/f259d10db577bbf1b11fffd028e678b38d5f402d/addons/stock/models/stock_picking.py#L1079)

Reply:
allora posso chiudere la pr a questo punto

Reply:
puoi aggiungere il `return` che male non fa!

Reply:
giusto stavo per scriverlo mi hai tolto le parole di bocca arrivo

Reply:
ci siamo